content/departments/product-engineering/product/process/unofficial_product_sales_metrics_definitions.md

```diff
@@ -0,0 +1,45 @@
+# Unofficial product sales metrics definitions
+
+While there are multiple company departments working to standardize and unify per-product revenue and sales reporting, that shouldn't stop us from evaluating the sales patterns of add-on products like Code Insights and Batch Changes. These metrics help us track success, prioritize investments, determine when parts of our product and sales strategy need iteration, and learn from what's working well.
+
+Since these numbers are somewhat unofficial due to our current sales process, the most important thing is that we use the same definitions so we can summarize and learn across products.
+
+(Of course, all the numeric values on this page are wildly fake and do not reflect neither proportionally nor absolutely the price or metrics of Sourcegraph. They exist for example purposes only.)
+
+## Attach rate
+
+The percentage of new logos that include an add-on product. When calculating attach rate on a quarterly basis, we do count expansion sales TODO THINK ABOUT THIS. When calculating overall, it's just the percentage of total customers with an add-on product.
```

Comment:
Major question here – how should we handle attach rate for expansion sales? Should we count those 1:1 like new logo lands? Should we just not count them in the per-quarter reporting? will give this some more thought with a fresher brain next week.